### ISSUE TITLE
S109: Magic number -1

### ISSUE THREAD

**Issue by nicolasbreuni** [SonarSource/SonarTS#896]
**I want to report a bug.**

**SonarTS version:** 2.1<br>
**Node.js version:** 10<br>
**TypeScript version:** 3.5.3<br>

**SonarQube version:** 8.0<br>
**SonarLint version and used IDE:** Intellij IDEA<br>
**TSLint version:** 5.20.1<br>

**Rule key:** S109

**Reproducer** 
```
// minimal reproducer if relevant
private readonly _id: number = -1; // No magic number: -1
```

**Expected behavior**

In the rule description -1, 0 and 1 are excluded numbers. Thus this line should not be reported.

![image](https://user-images.githubusercontent.com/42643976/71897607-5d923480-3157-11ea-8be9-059856120e51.png)
